packages/yoastseo/src/scoring/assessments/inclusiveLanguage/configuration/appearanceAssessments.js

```diff
@@ -12,18 +20,27 @@ const appearanceAssessments = [
 	{
 		identifier: "obese",
 		nonInclusivePhrases: [ "obese", "overweight" ],
-		inclusiveAlternatives: "<i>has/have a higher weight, " +
-			"higher-weight person/people, person/people in higher weight body/bodies, heavier person/people</i>",
+		inclusiveAlternatives: "<i>has a higher weight, " +
+			"higher-weight person, person in higher weight body, heavier person</i>",
+		score: SCORES.POTENTIALLY_NON_INCLUSIVE,
+		feedbackFormat: [ potentiallyHarmful, preferredDescriptorIfKnownPlusSize ].join( " " ),
```

Comment:
not sure why I didn't specify in the issue, but I think it also makes sense to use the `potentiallyHarmfulUnless` string instead of `potentiallyHarmful` for these phrases. If someone wants to refer to themselves as e.g. 'overweight' they have the right to do it.